qualification/pytest-jenkins.ini

```
@@ -0,0 +1,12 @@
+[pytest]
+tester = Tester one
```

Maybe something like
```suggestion
tester = Jenkins
```